evergreen/eg_app/consumers.py

```diff
@@ -177,9 +181,9 @@ def handle_post_upload(self, data):
                 return None
 
             caption = data.get("caption", "")
-            MAX_CHAR_LENGTH = 280
+            max_char_length = 280
 
-            if len(caption) > MAX_CHAR_LENGTH:
+            if len(caption) > max_char_length:
```

Comment:
Imo this and `MAX_IMAGE_SIZE` below make sense to keep in all caps as important constants that one may wish to change

Since afaik Py doesn't have true consts, is there a way to get the formatter to accept them in upper snake case? Like hoisting to the top of the file or something?

Reply:
Yes we should change that to be constants defined at the top of the file or perhaps somewhere in settings. I will need to ask @sumanyuj what those constants represent however.

Reply:
That was part of our demo protections: max text len (so no bee movie script) and max image size (so bee movie mp4)